packages/bench-trial/README.md

```diff
@@ -0,0 +1,94 @@
+# Bench trial
+
+> Runs one or multiple benchmark tests
+
+## Install
+
+```bash
+npm install -g bench-trial
+```
+
+## Usage
+
+```bash
+bench-trial benchmarks/map-helper-vs-entity.js -i 5
+```
+
+## TL;DR
+
+Runs one (or more) BenchmarkJs test multiple times enough to get less ambiguous results, includes basic testing to make sure tests are reliable. 
+
+## Why?
+
+While running [benchmarkjs](https://benchmarkjs.com) to compare different versions of code I found out a couple of things:
+
+- **Consistency**: I noticed that the same benchmark tests were returning different results every time they executed. If they were re-run consecutively, I would get more operations per second on each benchmark. I believe the reason may be related to the v8 engine warming up and optimizing the code the more it ran, since if I let some time to "cool off" the operations per second for each test would decrease. These ambiguous results meant having to repeat tests to ensure some consistency.
```

Comment:
The title should have stayed as `**Ambiguous Results**`. This is describing the problem.